libs/pika/execution/tests/unit/algorithm_drop_operation_state.cpp

```diff
@@ -165,5 +165,11 @@ int main()
 
     test_adl_isolation(ex::drop_operation_state(my_namespace::my_sender{}));
 
+    {
+        auto s = ex::just(std::tuple<int, double>()) | ex::drop_operation_state() |
+            ex::then([](std::tuple<int, double>&&) {});
+        tt::sync_wait(std::move(s));
+    }
+
```

Comment:
Would you mind restructuring this test in the same form as the tests from line 26 to 107 (you can move it up there as well). Maybe change one of the tuple types to `std::string` so we have one type where copy isn't the same as move?